src/energy_panel.py

```diff
@@ -132,7 +132,7 @@ def update_data(self, output):
             zval = ma.masked_array(self.hist2d[0])
             zval[zval == 0] = ma.masked
             zval *= float(zval.max())**(-1)
-            tmplist = [zval[~zval.mask].min(), zval.max()]
+            tmplist = [zval[not zval.mask].min(), zval.max()]
         else:
             zval = np.copy(self.hist2d[0])
             zval[zval==0] = 0.5
```

Comment:
not so sure about this one....to be honest feels like a bad bit of code, but probably best not to touch it.